examples/src/keyrings/aws_kms_mrk_keyring_example.py

```diff
@@ -0,0 +1,151 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""
+This example sets up the KMS MRK (multi-region key) Keyring
+
+KMS MRK keyring interacts with AWS Key Management Service (AWS KMS) to create, encrypt,
+and decrypt data keys using AWS KMS defined Customer Master Keys (CMKs).
+This example creates a KMS MRK Keyring and then encrypts a custom input EXAMPLE_DATA
+with an encryption context. This example also includes some sanity checks for demonstration:
+1. Ciphertext and plaintext data are not the same
+2. Encryption context is correct in the decrypted message header
+3. Decrypted plaintext value matches EXAMPLE_DATA
+These sanity checks are for demonstration in the example only. You do not need these in your code.
+
+AWS KMS MRK keyrings can be used independently or in a multi-keyring with other keyrings
+of the same or a different type.
+
+For more information on how to use KMS keyrings, see
+https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/use-kms-keyring.html
+
+For more info on KMS MRK (multi-region keys), see the KMS documentation:
+https://docs.aws.amazon.com/kms/latest/developerguide/multi-region-keys-overview.html
+"""
+import sys
+
+import boto3
+from aws_cryptographic_materialproviders.mpl import AwsCryptographicMaterialProviders
+from aws_cryptographic_materialproviders.mpl.config import MaterialProvidersConfig
+from aws_cryptographic_materialproviders.mpl.models import CreateAwsKmsMrkKeyringInput
+from aws_cryptographic_materialproviders.mpl.references import IKeyring
+from typing import Dict
+
+import aws_encryption_sdk
+from aws_encryption_sdk import CommitmentPolicy
+
+# TODO-MPL: Remove this as part of removing PYTHONPATH hacks.
+MODULE_ROOT_DIR = '/'.join(__file__.split("/")[:-1])
+
+sys.path.append(MODULE_ROOT_DIR)
+
+EXAMPLE_DATA: bytes = b"Hello World"
+
+
+def encrypt_and_decrypt_with_keyring(
+    encrypt_kms_key_id: str,
+    decrypt_kms_key_id: str,
+    encrypt_region: str,
+    decrypt_region: str
+):
+    """Demonstrate an encrypt/decrypt cycle using an AWS KMS keyring.
+
+    Usage: encrypt_and_decrypt_with_keyring(encrypt_kms_key_id,
+                                            decrypt_kms_key_id,
+                                            encrypt_region,
+                                            decrypt_region)
+    :param encrypt_kms_key_id: KMS Key identifier for the KMS key you want to use
+    for encryption of your data keys.
+    :type encrypt_kms_key_id: string
+    :param decrypt_kms_key_id: KMS Key identifier for the KMS key you want to use
+    for decryption of your data keys.
+    :type decrypt_kms_key_id: string
+    :param encrypt_region: AWS Region for encryption of your data keys
+    :type encrypt_region: string
+    :param decrypt_region: AWS Region for decryption of your data keys
+    :type decrypt_region: string
```

Comment:
Can you explicitly call out that these are expected to be the same KMS MRK, but in different regions?
The encrypt/decrypt_kms_key_ids don't explicitly say they're the same key.

Reply:
Do what you did on the other example, you have it right there